### ISSUE TITLE
impute_dtc and date-time object

### ISSUE THREAD

**Issue by slamao** [pharmaverse/admiral#175]
Hi @thomas-neitmann @kabis-ops , 
I am struggling to find out a correct call for `impute_dtc` with date-time string
```
library(lubridate)
now() %>% impute_dtc()
now() %>% as.character() %>% impute_dtc()
```
Both of these calls include standard date-time info but `impute_dtc` throws NA. 

According to documentation, the date-time should be provided in format "yyyy-mm-ddThh:mm:ss" which is different from the standard "yyyy-mm-dd hh:mm:ss". Do we have some function to convert from "yyyy-mm-dd hh:mm:ss" to "yyyy-mm-ddThh:mm:ss"? 
In other words, we have a function `convert_dtc_to_dtm`, do we have a function to convert in the opposite direction?
